src/node_buffer.cc

```diff
@@ -585,18 +585,19 @@ void Fill(const FunctionCallbackInfo<Value>& args) {
   }
 
   node::Utf8Value str(args.GetIsolate(), args[1]);
-  size_t str_length = str.length();
-  size_t in_there = str_length;
-  char* ptr = ts_obj_data + start + str_length;
+  const size_t str_length = str.length();
 
   if (str_length == 0)
     return;
 
+  char* ptr = ts_obj_data + start + str_length;
+
   memcpy(ts_obj_data + start, *str, MIN(str_length, length));
 
   if (str_length >= length)
     return;
 
+  size_t in_there = str_length;
```

Comment:
I'm saying if it already was how you have it in this PR and someone submitted a patch to group the variables together that the change is more stylistic than a cleanup or fix. Though maybe @bnoordhuis has an opinion on this.

I'm not worried about this either way. Just pointing out an observation.